src/Surfnet/ServiceProviderDashboard/Application/Metadata/OauthClientCredentialsClientJsonGenerator.php

```diff
@@ -49,13 +50,22 @@ public function __construct(
     ) {
     }
 
-    public function generateForNewEntity(ManageEntity $entity, string $workflowState): array
-    {
+    public function generateForNewEntity(
```

Comment:
Same here, see comment before... genereate what?

Reply:
This is a naming convention I use more regularly, the name of the class allready suggests it is a JSON generator. So generate is descriptive enough (to me)